english/train-ride/train-ride.md

```diff
@@ -0,0 +1,5 @@
+You are standing in front of a fire breathing, steam spitting locomotive. The Engineer grabs you by the collar and whisks you into the cab. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.